dotnet/src/SemanticKernel.Abstractions/Contents/StreamingChatMessageContent.cs

```diff
@@ -20,6 +20,11 @@ public class StreamingChatMessageContent : StreamingKernelContent
     /// </summary>
     public string? Content { get; set; }
 
+    /// <summary>
+    /// Name of the author of the message
+    /// </summary>
+    public string? Name { get; set; }
```

Comment:
Should this be marked as experimental as well?

Reply:
Absolutely!